TestFoundation/TestNSString.swift

```diff
@@ -1039,8 +1039,8 @@ func checkHasPrefixHasSuffix(lhs: String, _ rhs: String, _ stack: [UInt]) -> Int
     }
 
     var failures = 0
-    failures += testFailure(expectHasPrefix, lhs.hasPrefix(rhs), stack + [__LINE__])
-    failures += testFailure(expectHasSuffix, lhs.hasSuffix(rhs), stack + [__LINE__])
+    failures += testFailure(expectHasPrefix, lhs.hasPrefix(rhs), stack + [#line])
```

Comment:
This was fixed in a follow on patch.  Just put a space between the [ and # if you don't have a build with the fix yet.